src/Retriever.vue

```diff
@@ -59,10 +59,12 @@ export default {
             template: `<div>\n${result}\n</div>`,
           })
           new tempComponent().$mount(this.$el);
+          this.$emit('src-loaded');
```

Comment:
To let the parent panel know that the code is loaded / or failed to load so that we can get the `scrollHeight` for the animation to work.